topology/src/main/java/indexingTopology/util/shape/CheckInPolygon.java

```diff
@@ -0,0 +1,29 @@
+package indexingTopology.util.shape;
+
+/**
+ * Create by zelin on 17-11-29
+ **/
+public class CheckInPolygon implements CheckContain {
+
+    private Polygon polygon;
+    public CheckInPolygon() {
```

Comment:
You should write a member function in this class to allow the user specify the polygon.